assets/prototype/domain/eventEditor/containers/queries/useInitQueries.js

```diff
@@ -27,24 +25,18 @@ const useInitQueries = ({ eventId }) => {
 		},
 	});
 
-	if (!initialized) {
-		toaster.loading(loadingDates, toasterDatesMessage);
-		toaster.error(datetimeError);
-	}
-
 	const datetimes = pathOr([], ['datetimes', 'nodes'], datetimesData);
 	const datetimeIn = datetimes ? datetimes.map(({ id }) => id) : [];
-
 	const toasterTicketsMessage = 'initializing event editor tickets';
-	const [
-		fetchRelatedTickets,
-		{ data: ticketsData, error: ticketError, loading: loadingTickets, called },
-	] = useLazyQuery(GET_TICKETS, {
```

Comment:
From what I understand [from this Apollo post ](https://blog.apollographql.com/apollo-client-now-with-react-hooks-676d116eeae2)is intended to be used upon some user action, check `Lazy evaluation for queries` paragraph.
***
And based on this discussion: https://github.com/apollographql/apollo-feature-requests/issues/119
`skip` option seem more suitable for cases when query needs to be executed conditionally.
I might be missing something, please correct me, if so.